docker/dev/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM lychee-base:latest AS base
+
+# Install development dependencies
+
+RUN \
+	apt-get install -y git composer nodejs
```

Comment:
Unless I am wrong, you may end up with an old version of nodejs if you use bookworm as base.
https://github.com/nodesource/distributions#debian-and-ubuntu-based-distributions < what I use locally to get an updated version.

https://github.com/LycheeOrg/Lychee/blob/dockerfile/docker-context/Dockerfile see some work I had done a few weeks ago in that direction (line 25)

Reply:
great, I took it from the Lychee-Docker repo. but updated with your suggestion, using the node23 from nodesource

Reply:
The way it works on the Lychee-Docker repo is that we use a node image inside the docker file to compile the necessary files before pulling them to the base image, that way we don't have to deal with the nodejs layer.

Reply:
I understand what you mean, but this image is only for development, lets say you want to run some nodejs commands during the development inside the container. then we need the nodejs cli. For building production images ofc we dont need them, thats Why I split docker files into two. (base & dev)

Reply:
I know, I was just explaining why we didn't need the node source. while for the dev image, it is indeed necessary. :)